src/demo-app/tsconfig.json

```diff
@@ -22,8 +22,11 @@
       ]
     }
   },
-  "exclude": [
-    "*-aot.ts"
+  "files": [
+    "./demo-app-types.d.ts",
+    "./demo-app-module.ts",
+    "./system-config.ts",
+    "./main.ts"
```

Comment:
In the past WebStorm had issues when you specified `files` where it would only provide completion / error checking for _exactly_ these files and not their dependencies. Are you able to tell if that's still an issue?